docs/readme.chinese.md

```diff
@@ -104,269 +104,320 @@ Typescript 有两种 _不同_ 的类型结构: “类”和“泛型”。
 >
 > 有时候我们也会用“hk-type”来表达，这样其实更合适。
 
-## 1.4. 目录
-
-- [[HK-Types Toolbelt]](#--hk-types-toolbelt)
-  - [1.1. 安装](#11-安装)
-  - [1.2. 用法](#12-用法)
-    - [1.2.1. 用$引用高阶类（kind）](#121-用引用高阶类kind)
-    - [1.2.2. 子路径导入](#122-子路径导入)
-  - [1.3. HKT是什么?](#13-hkt是什么)
-  - [1.4. 目录](#14-目录)
-- [2. API](#2-api)
-  - [2.1. Basic Utilities](#21-basic-utilities)
-    - [2.1.1. $<F, X>](#211-f-x)
-    - [2.1.2. $$<FX, X>](#212-fx-x)
-    - [2.1.3. Cast<A, B>](#213-casta-b)
-  - [2.2. Boolean Types](#22-boolean-types)
-    - [2.2.1. Boolean.And\<X>](#221-booleanandx)
-    - [2.2.2. Boolean.Or\<X>](#222-booleanorx)
-    - [2.2.3. Boolean.Not](#223-booleannot)
-  - [2.3. Combinator Types](#23-combinator-types)
-    - [2.3.1. Combinator.Self](#231-combinatorself)
-    - [2.3.2. Combinator.ApplySelf](#232-combinatorapplyself)
-  - [2.4. Conditional Types](#24-conditional-types)
-    - [2.4.1. Conditional.Equals\<A>](#241-conditionalequalsa)
-    - [2.4.2. Conditional.Extends\<A>](#242-conditionalExtendsa)
-  - [2.5. Function Types](#25-function-types)
-    - [2.5.1. Function](#251-function)
-    - [2.5.2. Function.Constant\<A>](#252-functionconstanta)
-    - [2.5.3. Function.Identity](#253-functionidentity)
-  - [2.6. Kind Types](#26-kind-types)
-    - [2.6.1. Kind\<F>](#261-kindf)
-    - [2.6.2. Kind.Composable\<FX>](#262-kindcomposablefx)
-    - [2.6.3. Kind.Compose\<FX>](#263-kindcomposefx)
-    - [2.6.4. Kind.Pipe\<FX>](#264-kindpipefx)
-    - [2.6.5. Kind.\_](#265-kind_)
-  - [2.7. List Types](#27-list-types)
-    - [2.7.1. List.Map\<F>](#271-listmapf)
-    - [2.7.2. List.Find\<F>](#272-listfindf)
-    - [2.7.3. List.Filter\<F>](#273-listfilterf)
-    - [2.7.4. List.Append\<F>](#274-listappendf)
-    - [2.7.5. List.First\<T>](#275-listfirstt)
-    - [2.7.6. List.Last\<T>](#276-listlastt)
-    - [2.7.7. List.Pair\<T>](#277-listpairt)
-    - [2.7.8. List.Every\<T>](#278-listeveryt)
-    - [2.7.9. List.Some\<T>](#279-listsomet)
-    - [2.7.10. List.Reverse\<T>](#2710-listreverset)
-  - [2.8. String Types](#28-string-types)
-    - [2.8.1. String.StartsWith\<S>](#281-stringstartswiths)
-    - [2.8.2. String.EndsWith\<S>](#282-stringendswiths)
-    - [2.8.3. String.Includes\<S>](#283-stringincludess)
-    - [2.8.4. String.Append\<S>](#284-stringappends)
-    - [2.8.5. String.Prepend\<S>](#285-stringprepends)
-
-# 2. API
-
-柯里化函数下，这个仓库旨在用于使用 `point-free 编程风格` 来组合类型。在这方面，API 类型将首先接收 `执行的操作(一系列函数)` ，然后接收要操作的数据。
-
-> 先将会执行哪几个函数串起来，最后参数变量，连串贯通的意思。
-
-所有函数类 每次接受 _一个_ 参数，以支持 `柯里化函数` 和 `point-free 风格` 。
-
-> `point-free 风格` 解释: 基本上是你写代码但不显式地在代码中提供参数, 简单来说，就是省略函数的参数。
-
-这非常有用，特别是在需要函数的回调中, 让代码更易读。
-
-## 2.1. Basic Utilities
-
-### 2.1.1. $<F, X>
-
-`$` 操作符用于对定义类型更高类型的函数。  
-它相当于 TypeScript 中的 `F<A>` 语法。
+> **中文译者注**
+> 原文中出现的 higher-kinded type（高阶类）, type function（类函数）, higher-kinded-type function（高阶类函数）等词汇，虽然严格来说存在差别，但多数时候可以理解为相似的概念。
```

Comment:
These two lines aren't in the English readme, but were added to note a point regarding translation.

It basically says the terms _higher-kinded type_, _type function_ and _higher-kinded-type function_ can be thought of as similar for easier understanding.

If you don't like this, let me know, and I'm happy to remove it.

Reply:
I like this!